framework/src/components/storage/entities/block.js

```diff
@@ -22,29 +22,10 @@ const filterType = require('../utils/filter_types');
 const BaseEntity = require('./base_entity');
 const Transaction = require('./transaction');
 
-const defaultCreateValues = {};
-const createFields = [
-	'id',
-	'height',
-	'blockSignature',
-	'generatorPublicKey',
-	'payloadHash',
-	'payloadLength',
-	'numberOfTransactions',
-	'previousBlockId',
-	'timestamp',
-	'totalAmount',
-	'totalFee',
-	'reward',
-	'version',
-];
-
 const sqlFiles = {
 	select: 'blocks/get.sql',
 	count: 'blocks/count.sql',
-	create: 'blocks/create.sql',
 	isPersisted: 'blocks/is_persisted.sql',
-	delete: 'blocks/delete.sql',
 	getFirstBlockIdOfLastRounds: 'blocks/get_first_block_id_of_last_rounds.sql',
```

Comment:
This is chain related method as well, isn't it? If so, should be moved to `ChainBlock`